runtime/doc/news.txt

```diff
@@ -281,6 +281,8 @@ TERMINAL
   'scrollback' are not reflown.
 • The |terminal| now supports OSC 8 escape sequences and will display
   hyperlinks in supporting host terminals.
+• The |terminal| now updates the cursor shape and blink when set by an
```

Comment:
we should also mention `TermCursor` and `TermCursorNC` highlights currently don't work. maybe in the `:help news-breaking-dev` if we expect to fix this before 0.11 release.

Reply:
I fixed `TermCursor`, so we won't regress that in this PR (it was easy, just set it in `'guicursor'` and everything magically worked).

`TermCursorNC` will no longer be supported at all though. This is not a temporary regression, but a permanent change. When the terminal window doesn't have focus, there is simply no cursor, so there's no highlight group to apply. We still have `StatusLineTermNC` though.

I'll add this to `news.txt` and `deprecated.txt`.